Buildings/Air/Systems/SingleZone/VAV/BaseClasses/HysteresisWithHold.mo

```diff
@@ -0,0 +1,146 @@
+within Buildings.Air.Systems.SingleZone.VAV.BaseClasses;
+model HysteresisWithHold
+  extends Modelica.Blocks.Icons.Block;
+
+  parameter Real uLow = 0.05 "if y=true and u<=uLow, switch to y=false";
+  parameter Real uHigh = 0.15 "if y=false and u>=uHigh, switch to y=true";
+
+  parameter Modelica.SIunits.Time onHolDur=15*60
```

Comment:
Please remove from this model, and add them when you instantiate the model, as the values can change significantly between applications